johnsonandjohnson/Bodiless-JS
# feat(documentation): Support better control over IA

## Changes
- Modify documentation app 
  - Use js instead of json to define TOC's, enabling reuse of existing tocs.
  - Allow separation of docs into namespaces

## Test Instructions
- Run docs:build on site without a `bodiless.docs.toc.js` and verify that default structure is correct.
- Create a `bodiless.docs.toc.js` at site level and verify that it controls docs structure
- Create a package containing a namespaced `docs.json` file, update toc to refer to it, and verify that structure is correct.
